controllers/jitsi/jvb_controller.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/go-logr/logr"
+	"github.com/onmetal/meeting-operator/apis/jitsi/v1beta1"
+	meeterr "github.com/onmetal/meeting-operator/internal/errors"
+	"github.com/onmetal/meeting-operator/internal/jitsi"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+type JVBReconciler struct {
+	client.Client
+
+	Log    logr.Logger
+	Scheme *runtime.Scheme
+}
+
+func (r *JVBReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	return ctrl.NewControllerManagedBy(mgr).
+		For(&v1beta1.JVB{}).
+		WithEventFilter(r.constructPredicates()).
+		Complete(r)
+}
+
+func (r *JVBReconciler) constructPredicates() predicate.Predicate {
+	return predicate.Funcs{
+		UpdateFunc: onUpdate,
+	}
+}
+
+func onUpdate(e event.UpdateEvent) bool {
```

Comment:
`onUpdate` name may be confusing for filter function. Will it be better to use self-describing like `isSpecChanged` or `filterSpecChanged` etc..

Reply:
done